src/components/website/molecules/Header/Header.tsx

```diff
@@ -112,7 +120,7 @@ const Header = styled(
         </header>
         <Modal open={modalOpen} setOpen={setModalOpen}>
           <NavMobile>
-            <ListLinks links={links} />
+            <ListLinksMobile links={links} />
```

Comment:
Always mobile version ?

Reply:
I need to use NavMobileItem instead of NavItem.